notebooks/depth_time_series.ipynb

```diff
@@ -0,0 +1,2310 @@
+{
```

Comment:
<p>would it be easy to make this a histogram instead of a timeseries? It would be useful to see how the depth measurements are distributed through time before looking at the depth values themselves.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/iobis/pyobis/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Is this the kind of plot we are talking about?

![image](https://user-images.githubusercontent.com/36472216/182586086-6436d741-1b15-4bb8-99fc-0639f3f349b3.png)

Reply:
:raised_hands: This is much better!